app/scripts/components/dashboardComponents/listItems.jsx

```diff
@@ -136,6 +104,182 @@ class ListItems extends React.Component {
+                    <div style={styles.linkColor}>
+                        {this.iconPicker(child.kind)}
+                        {child.name.length > 82 ? child.name.substring(0, 82) + '...' : child.name}
+                        {child.kind === Kind.DDS_FILE ? ' (version '+ child.current_version.version+')' : ''}
+                    </div>
+                </a>
+            </TableRowColumn>
+        )
+    }
+    tableRowColumnLastUpdated(child, route, screenSize) {
+        if (screenSize && screenSize.width >= 680) {
+            let info = this.updatedOrCreatedInfo(child.audit)
+            if (child.kind === Kind.DDS_PROJECT) {
+                info = <a href={route} className="external" onClick={(e) => this.checkForAllItemsSelected(e)}>
+                    {info}
+                </a>
+            }
+            return (
+                <TableRowColumn onTouchTap={()=>this.check(child.id, child.kind)}>
+                    {info}
+                </TableRowColumn>
+            )
+        }
+    }
+    tableRowColumnSize(child, screenSize) {
+        if (screenSize && screenSize.width >= 840) {
+            let sizeInfo
+            if (child.kind === Kind.DDS_FILE && child.current_version) {
+                sizeInfo = BaseUtils.bytesToSize(child.current_version.upload.size)
+            } else {
+                sizeInfo = '---'
+            }
+            
+            if (child.kind === Kind.DDS_FILE) {
+                return (
+                    <TableRowColumn onTouchTap={() => this.check(child.id, child.kind)} style={{width: 100}}>
+                        {sizeInfo}
+                    </TableRowColumn>
+                )
+            } else {
+                return (
+                    <TableRowColumn onTouchTap={() => dashboardStore.selectItem(child.id, this.props.router)} style={{width: 100}}>
+                        {sizeInfo}
+                    </TableRowColumn>
+                )
+            }
+        }
+    }
+    tableRowColumnMenu(child, menuWidth) {
+        let optionsMenu
+        if (child.kind === Kind.DDS_FILE) {
+            optionsMenu = <FileOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FILE, true)}/>
+        } else if (child.kind === Kind.DDS_FOLDER) {
+            optionsMenu = <FolderOptionsMenu {...this.props} clickHandler={()=>this.setSelectedEntity(child.id, Path.FOLDER, true)}/>
+        } else if (child.kind === Kind.DDS_PROJECT) {
+            optionsMenu = <ProjectOptionsMenu {...this.props} clickHandler={()=>mainStore.setSelectedProject(child.id)}/>
+        }
+        return (
+            <TableRowColumn style={{textAlign: 'right', width: menuWidth}}>
+                <div onClick={(e) => {e.stopPropagation()}}>
+                    {optionsMenu}
+                </div>
+            </TableRowColumn>
+        )
+    }
+  
+    // HelperFunctions
+    listItemRoute(child, componentName) {
+        if (componentName == 'Dashboard') {
```

Comment:
Isn't it called Navigator now?

Reply:
Also, make sure to use `===` and not `==` or you might get unwanted type coercion. 

For example 
```javascript
42 == "42" // true
42 === "42" // false
```

Reply:
Good catch, I guess I fixed that during an earlier refactoring, and I didn't find any other places I used `==` so 👍